Source/Scene/ModelExperimental/Model3DTileContent.js

```diff
@@ -384,7 +393,9 @@ Model3DTileContent.fromGeoJson = function (tileset, tile, resource, geoJson) {
     content,
     additionalOptions
   );
-  content._model = ModelExperimental.fromGeoJson(modelOptions);
+  const model = ModelExperimental.fromGeoJson(modelOptions);
+  content._model = model;
+  content._readyPromise = model.readyPromise;
   return content;
```

Comment:
```suggestion
  content._readyPromise = model.readyPromise;
  
  return content;
```
one more nitpick for consistency with the other `fromXXX` functions